nekoyume/Assets/StreamingAssets/Localization/common.csv

```diff
@@ -634,6 +634,7 @@ UI_MAIN_MENU_COMBINATION,Workshop,공방,Oficina,,ワークショップ,Werkstat
 UI_MAIN_MENU_QUEST,Stage!,스테이지!,Fase!,,ステージ！,Ebene!,,ด่าน!,Poziom!,,,Stage!,Niveau !,,,,层级！,
 UI_MAIN_MENU_RANKING,Arena,아레나,Arena,,アリーナ,Arena,Arena,อารีน่า,Arena,Arena,Arena,Arena,Arène,Арена,សៃវ្យេន,,竞技场,
 UI_MAIN_MENU_SHOP,Shop,상점,Comprar,,ショップ,Geschäft,Tienda,ร้านค้า,Sklep,Parduotuvė,Winkel,Toko,Boutique,Магазин,ហាង,,店铺,
+UI_MAIN_MENU_STAKING,Staking,스테이킹,,,,,,,,,,,,,,,,
```

Comment:
I think this lable overlap [`UI_PLACE_STAKING`](https://github.com/planetarium/NineChronicles/blob/7d540098cd6674fcb8a82518c02460f22cf86d10/nekoyume/Assets/StreamingAssets/Localization/common.csv#L1059).

Reply:
I will replace with your suggestion. but I have a question, does the existing name of monster collecting stay the same?